themes/wiki/.htaccess

```
@@ -1 +1 @@
-Redirect 301 /wiki/ https://wiki.freecadweb.org/
+Redirect 301 /wiki/ https://Wiki.FreeCAD.Org/
```

Rest of the page has lowercase domain names and server converts it, best to stick to the convention here
```suggestion
Redirect 301 /wiki/ https://wiki.freecad.org/
```